tests/unit/test_client.py

```diff
@@ -1704,14 +1713,16 @@ def _download_blob_to_file_helper(
             blob._CHUNK_SIZE_MULTIPLE = 1
             blob.chunk_size = 3
         blob._do_download = mock.Mock()
-
         file_obj = io.BytesIO()
-        if raw_download:
-            client.download_blob_to_file(
-                blob, file_obj, raw_download=True, **extra_kwargs
-            )
-        else:
-            client.download_blob_to_file(blob, file_obj, **extra_kwargs)
+        with patch.object(
```

Comment:
@Breathtender can you test that the invocation id changes for each call of download_blob_to_file()?

Reply:
Alright, check now; lines 1778-1795 in this file.